man/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ list(APPEND MAN_ALIAS
 	fido_assert_new fido_assert_authdata_ptr
 	fido_assert_new fido_assert_clientdata_hash_len
 	fido_assert_new fido_assert_clientdata_hash_ptr
+	fido_assert_new fido_assert_blob_len
+	fido_assert_new fido_assert_blob_ptr
```

Comment:
sort

Reply:
fixed in 105034ed49b1f6c2c2b8c3e32b770366a911625f